holidays/countries/yemen.py

```
@@ -0,0 +1,207 @@
+    # %s (estimated).
+    estimated_label = tr("%s (مُقدَّر)")
+    # %s (observed).
+    observed_label = tr("%s (ملاحظة)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (مُقدَّر ملاحظة)")
+    # The Republic of Yemen was declared on 22 May 1990.
+    start_year = 1991
+    supported_categories = (PUBLIC, SCHOOL, WORKDAY)
+    supported_languages = ("ar", "en_US")
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
```

Pylint (3.3.7)

[warning] Keyword argument before variable positional arguments list in the definition of __init__ function

(W1113)